Research/Research.xcodeproj/project.pbxproj

```diff
@@ -3344,7 +3345,7 @@
 				DEFINES_MODULE = YES;
 				DEVELOPMENT_TEAM = KA9Z8R6M6K;
 				DYLIB_COMPATIBILITY_VERSION = 1;
-				DYLIB_CURRENT_VERSION = 30;
+				DYLIB_CURRENT_VERSION = 29;
```

Comment:
hopefully not important and will get resolved with next stable release, but why is this going backward

Reply:
huh